### PR TITLE
Initial cleanup for 1.0.

### PR DESCRIPTION
**Pull request by portsmouth**
Add direct link to parameter reference from README.

### CODE REVIEW

**Comment by portsmouth** [comment]
Clarified front page:

image

**Comment by portsmouth** [comment]
The spec takes about 10 seconds to properly render, maybe we should mention that next to the white paper link?

**Comment by portsmouth** [comment]
Added a copy-pastable BibTeX citation reference, which is needed if we want OpenPBR to be referenced in papers or documentation. (The form of this could be tweaked, we can check how it renders in a paper for example).

```
@techreport{OpenPBR,
  author = {Zap Andersson and Paul Edmondson and Julien Guertault and Adrien Herubel and Alan King and Peter Kutz and Andréa Machizaud and Jamie Portsmouth and Frédéric Servant},
  title = {OpenPBR Surface},
  institution = {Academy Software Foundation (ASWF)},
  year = {2024},
  url = {https://academysoftwarefoundation.github.io/OpenPBR/}
}
```

**Comment by portsmouth** [comment]
Added a clickable GitHub icon to the spec (just after the version info), which links back to the GitHub page.

image

**Comment by portsmouth** [comment]
The reference to the Gulbrandsen paper is broken.

**Comment by portsmouth** [comment]
Rewording the text on the front page, as the sentence was long and hard to parse.

>OpenPBR Surface is a specification of a standard surface shading model for computer graphics. It aims to provide a material representation capable of accurately modeling the vast majority of CG materials used in practical visual effects and feature animation productions.

File: README.md [hunk 1]
```diff
@@ -6,15 +6,17 @@
...
-OpenPBR is an open standard hosted by the [Academy Software Foundation](https://www.aswf.io/) (ASWF), and is organized as a subproject of [MaterialX](https://materialx.org/).
+OpenPBR Surface is a specification of a surface shading model intended as a standard for computer graphics. It aims to provide a material representation capable of accurately modeling the vast majority of CG materials used in practical visual effects and feature animation productions.
 
+OpenPBR Surface is an open standard hosted by the [Academy Software Foundation](https://www.aswf.io/) (ASWF), and is organized as a subproject of [MaterialX](https://materialx.org/).
 
-### Resources
+### Specification
 
-* **[Specification](https://academysoftwarefoundation.github.io/OpenPBR/)**
+* **[White paper](https://academysoftwarefoundation.github.io/OpenPBR/)**
```
Comment (virtualzavie) [inline README.md:16]: This line reverts the changes from #182. Is this intentional or an oversight?

File: README.md [hunk 2]
```diff
@@ -6,15 +6,17 @@
...
+* **[Parameter reference](https://academysoftwarefoundation.github.io/OpenPBR/#parameterreference)**
 * **[Reference implementation](reference/open_pbr_surface.mtlx)** – written in [MaterialX](https://materialx.org/)
+* **[BibTeX citation](citation.bib)**
```
Comment (virtualzavie) [inline README.md:19]: 👍
Minor remark: I would have named it `openpbr.bib`, as a convenience for people downloading the file.

File: index.html [hunk 3]
```diff
@@ -2,22 +2,24 @@
 <link rel="stylesheet" href="style/style.css">
 <meta name="viewport" content="width=device-width,initial-scale=1">
 
-**OpenPBR Surface specification**
+**OpenPBR Surface**
```
Comment (virtualzavie) [inline index.html:5]: I question this change.
I think it's useful to see in the title that the document is a specification, especially considering that we may have more documents in the future, like an artist guide for example.

File: index.html [hunk 4]
```diff
@@ -2,22 +2,24 @@
...
 
 <center>Zap Andersson, Paul Edmondson, Julien Guertault, Adrien Herubel, Alan King, </center>
 <center>Peter Kutz, Andréa Machizaud, Jamie Portsmouth, Frédéric Servant.</center>
 
-<center>*Version 0.4, 2024-04-30. A joint work of Adobe and Autodesk*</center>
+<center>*Version 0.4, 2024-04-30. A joint work of Adobe and Autodesk &nbsp; <a href="https://github.com/AcademySoftwareFoundation/OpenPBR" title="Redirect to GitHub page"><img src="images/github-mark.png" alt="homepage" width="24" style="vertical-align: middle; position: relative; bottom: 3px"/></a> *</center>
```
Comment (virtualzavie) [inline index.html:10]: I see the value of having a link in the document, but I'm not a fan of a using a logo for that link.
I think a full text link would be clearer and look cleaner as well.

File: index.html [hunk 5]
```diff
@@ -1397,6 +1399,8 @@
 
 [#Gritz2010]: Larry Gritz, Clifford Stein, Chris Kulla, and Alejandro Conty. *Open Shading Language*, ACM SIGGRAPH Talks (2010).
 
+[#Gulbrandsen2014]: Ole Gulbrandsen. *Artist Friendly Metallic Fresnel*, Journal of Computer Graphics Techniques (2014).
```
Comment (virtualzavie) [inline index.html:1402]: 👍

File: citation.bib [hunk 6]
```diff
@@ -0,0 +1,7 @@
+@techreport{OpenPBR,
+  author = {Zap Andersson and Paul Edmondson and Julien Guertault and Adrien Herubel and Alan King and Peter Kutz and Andréa Machizaud and Jamie Portsmouth and Frédéric Servant},
+  title = {OpenPBR Surface},
+  institution = {Academy Software Foundation (ASWF)},
+  year = {2024},
```
Comment (virtualzavie) [inline citation.bib:5]: I'd suggest to add `version` and `month` fields, as we may have subsequent versions published the same year.

**Comment by virtualzavie** [comment]
> Someone with admin access is probably needed to do that, as it may require pointing the GitHub Pages link to the `internal/index.html` etc. See the dialog below for that.

Internal doesn't seem like a faithful description of the nature of those files. Accessing them directly is only problematic on GitHub, but works fine for someone who clones the repository.
What about `specification/index.html` or `spec/index.html`? This way they are moved away from top-level, but are not "hidden".

File: index.html (hunk 3 above)
Comment (virtualzavie) [reply index.html:5]: Note that "specification" and the "version" string could be moved to a sub-title before the author list.

**Comment by virtualzavie** [comment]
> The spec takes about 10 seconds to properly render (due to all the MathJax I assume), maybe we should mention that next to the white paper link?

On my side, it loads nearly instantaneously, both on Firefox/macOS, Firefox/Windows and Edge/Windows.

File: README.md (hunk 1 above)
Comment (portsmouth) [reply README.md:16]: I think it's actually better like this, as under the "Specification" heading we have a number of components, i.e. the white paper, the parameter reference, the MaterialX graph, and the citation.

We say now "OpenPBR surface is a specification" (rather than e.g. "A white paper specifying" as for standard surface). But there is a white paper as part of that specification. Hopefully it makes sense..

File: README.md (hunk 2 above)
Comment (portsmouth) [reply README.md]: Done in 5244f958157f855e51980e8d4098e5e87c51cfd3

File: README.md (hunk 1 above)
Comment (virtualzavie) [reply README.md:16]: Oh ok, I see the logic.

File: index.html (hunk 3 above)
Comment (portsmouth) [reply index.html:5]: How about this? (Added in c81366e84ffbba7dfbdd5ca358b1640e51736657)

image

File: index.html (hunk 4 above)
Comment (portsmouth) [reply index.html]: Really? I find the GitHub logo fairly universal, and a neat way to indicate what the link is.

If we make it a text link, what is the text, and where.

**Comment by portsmouth** [comment]
> > The spec takes about 10 seconds to properly render (due to all the MathJax I assume), maybe we should mention that next to the white paper link?
> 
> On my side, it loads nearly instantaneously, both on Firefox/macOS, Firefox/Windows and Edge/Windows.

Do the equations render correctly nearly instantaneously?  On my system, they show up immediately, but change format as they are correctly rendered.

File: index.html (hunk 3 above)
Comment (virtualzavie) [reply index.html:5]: Yep, I think that works.
(except of course for the additional changes mentioned in the meeting)

File: index.html (hunk 4 above)
Comment (virtualzavie) [reply index.html]: Matter of taste I guess; that's a cosmetic detail anyway, let's move on to the rest.

File: openpbr.bib [hunk 7]
```diff
@@ -3,5 +3,7 @@ @techreport{OpenPBR
   title = {OpenPBR Surface},
   institution = {Academy Software Foundation (ASWF)},
   year = {2024},
+  month = {May},
+  version = {1.0},
```
Comment (virtualzavie) [inline openpbr.bib:7]: A note of caution to @jstone-lucasfilm for the release candidate, that we now have a third place where the version number appears.

Comment (portsmouth) [reply openpbr.bib:7]: I'm not convinced a version number is needed in a citation. It can be assumed that the reference will want to refer to the latest version, which is available at the given link.

**Comment by portsmouth** [comment]
> > Someone with admin access is probably needed to do that, as it may require pointing the GitHub Pages link to the `internal/index.html` etc. See the dialog below for that.
> 
> Internal doesn't seem like a faithful description of the nature of those files. Accessing them directly is only problematic on GitHub, but works fine for someone who clones the repository. What about `specification/index.html` or `spec/index.html`? This way they are moved away from top-level, but are not "hidden".

Sounds fine to me. But I can't make this change myself as it will break the GitHub pages deployment. @jstone-lucasfilm would you mind to make this change?

File: openpbr.bib (hunk 7 above)
Comment (jstone-lucasfilm) [reply openpbr.bib:7]: Agreed that the version number is not usually included in citations, so this may not end up being an issue in practice.

**Comment by jstone-lucasfilm** [comment]
@portsmouth For moving the specification to a new location, I'm happy to assist, but I'd recommend that we leave this as an improvement for after today's release candidate.

File: index.html (hunk 3 above)
Comment (jstone-lucasfilm) [reply index.html:5]: Should we include the removal of top-level company and author names as part of this changelist, as suggested in today's meeting?

**Comment by portsmouth** [comment]
> @portsmouth For moving the specification to a new location, I'm happy to assist, but I'd recommend that we leave this as an improvement for after today's release candidate.

Sure that's fine. How about we merge the other improvements in this PR, and leave that for a different PR?

**Comment by jstone-lucasfilm** [comment]
@portsmouth Sure, I'm fine for this PR to be the first in a series of improvements over the next two weeks, but then maybe the title of the PR should be more specific?  :D

File: openpbr.bib (hunk 7 above)
Comment (portsmouth) [reply openpbr.bib:7]: I'll remove it for now.

File: index.html (hunk 3 above)
Comment (portsmouth) [reply index.html:5]: OK let me try it.

Comment (portsmouth) [reply index.html:5]: >removal of top-level company 

Do we want to mention ASWF in its place?

**Comment by portsmouth** [comment]
> @portsmouth Sure, I'm fine for this PR to be the first in a series of improvements over the next two weeks, but then maybe the title of the PR should be more specific? :D

OK, changed the PR title.

How is this? The ASWF logo doubles as the link back to the OpenPBR GitHub, killing two birds with one stone.

image


I moved the authors to the end, and took the liberty of moving @jstone-lucasfilm to that list, as that seems fairer to me considering your contribution. The BibTeX citation is updated accordingly, with the version removed.

image

```
@techreport{OpenPBR,
  author = {Zap Andersson and Paul Edmondson and Julien Guertault and Adrien Herubel and Alan King and Peter Kutz and Andréa Machizaud and Jamie Portsmouth and Frédéric Servant and Jonathan Stone},
  title = {OpenPBR Surface},
  institution = {Academy Software Foundation (ASWF)},
  year = {2024},
  month = {May},
  url = {https://academysoftwarefoundation.github.io/OpenPBR/}
}
```

**Comment by portsmouth** [comment]
Let me add the shader playground image as well...

**Comment by virtualzavie** [comment]
I think we can thank a few more people who have made significant contributions in the last months.
Chris Kulla, Eugene d'Eon, Masuo Suzuki, Andrea Weidlich, André Mazzone and Stephen Hill come to mind.

File: index.html [hunk 8]
```diff
@@ -1407,7 +1404,20 @@
...
+Paul Edmondson,
+Julien Guertault,
+Adrien Herubel,
+Alan King,
+Peter Kutz,
+Andréa Machizaud,
+Jamie Portsmouth,
+Jonathan Stone
+and
+Frédéric Servant.
```
Comment (virtualzavie) [inline index.html:1418]:
```suggestion
Frédéric Servant
and
Jonathan Stone.
```

File: index.html [hunk 9]
```diff
@@ -2,22 +2,19 @@
 <link rel="stylesheet" href="style/style.css">
 <meta name="viewport" content="width=device-width,initial-scale=1">
 
-**OpenPBR Surface specification**
+**OpenPBR Surface**
 
-<center>Zap Andersson, Paul Edmondson, Julien Guertault, Adrien Herubel, Alan King, </center>
-<center>Peter Kutz, Andréa Machizaud, Jamie Portsmouth, Frédéric Servant.</center>
+<center>*Specification v0.4, 2024-04-30. &nbsp; &nbsp; <a href="https://github.com/AcademySoftwareFoundation/OpenPBR" title="Redirect to GitHub page"><img src="images/aswf-logo-black.svg" alt="homepage" height="18" style="vertical-align: middle; position: relative; bottom: 3px"/></a> *</center>
```
Comment (virtualzavie) [inline index.html:7]:
```suggestion
<center>*Specification v0.4, 2024-05-14. &nbsp; &nbsp; <a href="https://github.com/AcademySoftwareFoundation/OpenPBR" title="Redirect to GitHub page"><img src="images/aswf-logo-black.svg" alt="homepage" height="18" style="vertical-align: middle; position: relative; bottom: 3px"/></a> *</center>
```

**Comment by portsmouth** [comment]
How's this?

image

**Comment by portsmouth** [comment]
> I think we can thank a few more people who have made significant contributions in the last months. Chris Kulla, Eugene d'Eon, Masuo Suzuki, Andrea Weidlich, André Mazzone and Stephen Hill come to mind.

Good point, thanks for the list. I updated it to:

image

**Review by jstone-lucasfilm — APPROVED** [review]
This looks good to me, thanks @portsmouth.

File: index.html (hunk 9 above)
Comment (virtualzavie) [reply index.html]: Ok that change was not actually useful, since the version hasn't been bumped yet. Sorry about that. 🙃
